src/pages/users/register/RegisterCompleted.jsx

```
@@ -0,0 +1,27 @@
+
+  return (
+    <div className={styles.registerContainer}>
+      <Stepper currentStep={step} />
+
+      <h1>
+        <b>회원가입</b>이 완료되었습니다.
+      </h1>
+
+      <Button
+        children="로그인 하기"
+        onClick={() => {
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)